shared/constants/keyring.ts

```diff
@@ -8,10 +8,19 @@ export enum InternalKeyringType {
   imported = 'Simple Key Pair',
 }
 
+///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+export enum SnapKeyringType {
+  snap = 'Snap Keyring',
```

Comment:
Enum keys should be PascalCase.
```suggestion
  Snap = 'Snap Keyring',
```

Reply:
This needs to be refactored for the entire keyring enum I think, perhaps out of scope

Reply:
The lower case used to follow the pattern used in the other keyrings.